source/index.txt

```diff
@@ -29,25 +26,92 @@ MongoDB Kafka Connector
 Overview
 --------
 
-The `MongoDB Kafka connector <https://www.mongodb.com/kafka-connector>`__ is
-a Confluent-verified connector that persists data from Kafka topics as a
+The `{+connector-long+} <https://www.mongodb.com/kafka-connector>`__ is
+a Confluent-verified connector that persists data from {+kafka+} topics as a
 data sink into MongoDB as well as publishes changes from MongoDB into Kafka
 topics as a data source.
 
-This guide provides information on available configuration options and
-examples to help you complete your implementation in the following
-sections:
-
-- :doc:`What's New </whats-new>`
-- :doc:`Quick Start </quick-start>`
-- :doc:`Introduction </introduction>`
-- :doc:`Tutorials </tutorials>`
-- :doc:`Sink Connector </sink-connector>`
-- :doc:`Source Connector </source-connector>`
-- :doc:`Security and Authentication </security-and-authentication>`
-- :doc:`Monitoring </monitoring>`
-- :doc:`Migrate from the Community Connector </migrate-from-kafka-connect-mongodb>`
-- :doc:`Troubleshooting </troubleshooting>`
-- :doc:`How to Contribute </contribute>`
-- :doc:`Issues & Help </issues-and-help>`
-- :doc:`Compatibility </compatibility>`
+What's New
+----------
+
+For a list of new features and changes in each version, see the
+:ref:`kafka-connector-whats-new` section.
+
+Quick Start
+-----------
+
+Learn how to get started with the {+connector+} and begin working with data
+in the :ref:`kafka-quick-start` section.
+
+Introduction
+------------
+
+Learn about and how the {+connector+} passes data between {+kafka+}
+and MongoDB in the :ref:`kafka-connector-introduction` section. In this
+section, you can also learn about how to install the connector and
+configure it to connect to your MongoDB deployment.
+
+Tutorials
+---------
+
+Follow :ref:`tutorials <kafka-connector-tutorials>` to learn how to set
+up the {+connector+} to read and write data for several use cases.
+
+Sink Connector
+--------------
+
+Learn how to configure how the {+sink-connector+} writes data from
+{+kafka+} into MongoDB in the :ref:`kafka-sink-overview` section.
+
+Source Connector
+----------------
+
+Learn how to configure how the {+source-connector+} writes data from
+MongoDB into {+kafka+} in the :ref:`kafka-source-overview` section.
+
+Security and Authentication
+---------------------------
+
+Learn how to secure communications between MongoDB and the
+connector in the :ref:`kafka-connector-security-auth` section. In this
+section, you can also learn how to authenticate with MongoDB by using
+AWS-based authentication.
```

Comment:
Suggestion:

I think this could statement could be more informative by identifying what is authenticating with MongoDB and "AWS-based authentication" are.

```suggestion
section, you can also learn how to configure the {+connector+} to connect to MongoDB using AWS IAM credentials.
```

I hope that's not too specific such that it won't need to change soon. I'm not aware of any other methods of authentication for the connector that are planned.